lib/util/index.js

```diff
@@ -591,6 +580,44 @@ function createMediaSignalingPayload(dominantSpeaker, networkQuality, trackSwitc
     );
 }
 
+/**
+ * Create {@link VideoTrack.Dimensions} RSP payload.
+ * @param {VideoTrack.Dimensions} [dimensions]
+ * @returns {object}
+ */
+function createDimensionsPayload(dimensions) {
+  return ['height', 'width'].reduce((payload, dimension) => {
+    const value = dimensions[dimension];
+    if (typeof value === 'number') {
+      payload = Object.assign({
+        [dimension]: value
+      }, payload);
+    }
+    return payload;
+  }, {});
+}
+
+/**
+ * Create {@link VideoRenderDimensions} RSP payload.
+ * @param renderDimensions
+ * @returns {object}
+ */
+function createRenderDimensionsPayload(renderDimensions) {
+  return [
+    ['dominantSpeaker', 'active_speaker']
```

Comment:
since `dominantSpeaker` is the only one that has `payloadProp != prop`  - I would take it out and make the rest of the processing bit easier to read.

Reply:
@makarandp0 I think this is fairly easy to read because we've set the initial value of `payloadProp` to `prop` in the `reduce` callback, which means that if the object prop and RSP prop are the same, then `payloadProp` will be equal to `prop`.